Makefile

```diff
@@ -264,10 +264,11 @@ e2e_symfony:		 ## Packages a fresh Symfony app
 e2e_symfony: fixtures/build/dir012/vendor box
 	composer dump-env prod --working-dir fixtures/build/dir012
 
-	APP_ENV=prod APP_DEBUG=0 php fixtures/build/dir012/bin/console --version > fixtures/build/dir012/expected-output
+	php fixtures/build/dir012/bin/console --version > fixtures/build/dir012/expected-output
 	rm -rf fixtures/build/dir012/var/cache/prod/*
 
-	APP_ENV=prod APP_DEBUG=0 ./box compile --working-dir fixtures/build/dir012
+	php fixtures/build/dir012/bin/console cache:clear
```

Comment:
This is the actual fix. Somehow `post-autoload-dump` with a `cache:clear` does not work

Reply:
>Somehow post-autoload-dump with a cache:clear does not work

??? What kind of error is it causing? That might be a big issue :/

Reply:
```
The command "'/usr/local/bin/composer' 'dump-autoload' '--classmap-authorit  
  ative' '--no-dev'" failed.                                                   
                                                                               
  Exit Code: 104(Unknown error) 
```

Reply:
What should we do here?